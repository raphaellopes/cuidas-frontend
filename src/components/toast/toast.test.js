// vendors
import React from 'react';
import renderer from 'react-test-renderer';

// locals
import Toast from './index';

const component = props => (renderer.create(
  <Toast {...props} />,
));

describe('<Toast />', () => {
  test('Should render the component', () => {
    const tree = component({ children: 'My mesage' }).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('Should dipatch onTimeout when the prop is set', () => {
    const tree = component({
      children: 'My mesage',
      onTimeout: () => ({}),
    }).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
