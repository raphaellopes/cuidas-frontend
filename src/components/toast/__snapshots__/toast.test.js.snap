// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Toast /> Should dipatch onTimeout when the prop is set 1`] = `
<div
  className="sc-bwzfXH dlrwnW"
>
  <div
    className="sc-htpNat kvZYzn"
  >
    <i
      className="icon fa fa-check-circle sc-bdVaJa jToUlD"
    />
  </div>
  My mesage
</div>
`;

exports[`<Toast /> Should render the component 1`] = `
<div
  className="sc-bwzfXH dlrwnW"
>
  <div
    className="sc-htpNat kvZYzn"
  >
    <i
      className="icon fa fa-check-circle sc-bdVaJa jToUlD"
    />
  </div>
  My mesage
</div>
`;
